### DISCUSSION TITLE
Merge multiple text files - not overwrite

### DISCUSSION THREAD

**Discussion by PSIG-dev** [WinMerge/winmerge#1991] [Q&A] [answered]
Hello!  Trying WinMerge for the first time, and cannot accomplish this.

I wish to merge 3 text files, combining the different text in each.  Example - 3 students are asked where to go on a school field trip.  The student responses are 3 text files with _one word_ each:
Kali's text file says "Museum"
Rolf's says "Park"
Antony's says "Circus"

I wish to output a merge file that lists:
Museum
Park
Circus

My attempts only result in one text overwriting the other.  Tutorial videos and docs stop short of blending and output.  How do I accomplish this simple merge task?  Thanks!

**Comment by sdottaka** [comment, marked as answer]
Unfortunately WinMerge doesn't have the ability to do that easily.
To do that in WinMerge you have to copy and paste repeatedly.

**Comment by PSIG-dev** [comment]
**Thank you** @sdottaka for the simple answer - it can't, easily.  And while I'm an easy-going type, I have to say that a simple **_merge_** like this is why I tried a program called Win**_Merge_**.  A simple statement in the description that it will not do simple **_merges_** may save many that assume too much, a great deal of time.

If anyone has suggestions for a lite program that can accomplish this "simple" task, I'm all ears!
